src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/main.js

```diff
@@ -171,9 +170,7 @@ define([
                     el: $container,
                     url: this.options.url,
                     instanceName: this.options.instanceName,
-                    preSelectedIds: _.map(this.getData().ids, function(id) {
-                        return {id: id};
-                    }),
+                    preselected: this.getItems(),
```

Comment:
From what I see `getItems` returns the same as the previous `map`, why did the name of the property change from `preSelectedIds` to `preselected` then?

Reply:
there is no option `preSelectedIds` in media overlay its just `preselected`: https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js#L52